pkgs/applications/networking/iptraf-ng/default.nix

```
@@ -1,24 +1,25 @@
-{ stdenv, fetchurl, ncurses }:
+{ stdenv, fetchFromGitHub, ncurses }:
 
 stdenv.mkDerivation rec {
-  version = "1.1.4";
+  version = "v1.2.1";
```

The version should be 

```suggestion
  version = "1.2.1";
```

See